### DISCUSSION TITLE
Rosetta 2 notice displayed when installing GPXSee (even on Intel Mac)

### DISCUSSION THREAD

**Discussion by markchagers** [Homebrew/discussions#5396] [Casks]
### Output of `brew config`

```shell
HOMEBREW_VERSION: 4.3.1
ORIGIN: https://github.com/Homebrew/brew
HEAD: e0bc557e7b991cb23583679e1cf1c8a92b793aeb
Last commit: 4 days ago
Core tap JSON: 24 May 07:10 UTC
Core cask tap JSON: 24 May 07:10 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 3.3.1 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/3.3.1/bin/ruby
CPU: quad-core 64-bit kabylake
Clang: 15.0.0 build 1500
Git: 2.45.1 => /usr/local/bin/git
Curl: 8.4.0 => /usr/bin/curl
macOS: 13.6.7-x86_64
CLT: 15.1.0.0.1.1700200546
Xcode: N/A
```

### Output of `brew doctor`

```shell
Your system is ready to brew.
```

### Description of issue

When installing GPXSee cask, the following message is displayed, even on Intel Macs:
`gpxsee is built for Intel macOS and so requires Rosetta 2 to be installed.`
Additionally the installed app cannot be started with MacOS complaining that it can't check it for malicious software. 

<img width="372" alt="image" src="https://github.com/Homebrew/discussions/assets/529428/3c477bd4-da61-4071-b94c-24eddd8823fd">

When installed with the --no-quarantine option it runs normally (at least on Intel Mac).

### Relevant casks

gpxsee

**Comment by p-linnane** [comment]
The message about Rosetta is a [`caveat`](https://docs.brew.sh/Cask-Cookbook#stanza-caveats). It is displayed regardless of your architecture.

The other issue you are seeing is macOS' Gatekeeper warning you that the application is not properly signed. This is not a Homebrew issue. The authors of GPXSee need to correctly sign their macOS releases to prevent this.

**Comment by tumic0** [comment]
TLDR: use MacPorts instead of Homebrew

The "GPXSee on macOS" situation is sad but reflects the user interest. As there is no real user interest (no one is willing to pay for Apple HW or the Apple "racketeering" fee for the certificates) there is no official ARM/Apple signed build of GPXSee. The situation in Homebrew is the [same](https://github.com/Homebrew/homebrew-cask/issues/159352) - there is no interest to provide at least an ARM build to the users. So all you can get for free at the moment is the MacPorts build which is also recommended on the GPXSee homepage. You may also use the GPXSee [Qt6 CI build](https://github.com/tumic0/GPXSee/actions/runs/9227617599) that is compiled for both Intel and ARM, but nothing will be signed by Apple.

**Reply by p-linnane** [reply]
Regardless of your feelings on the matter, applications will eventually need to be signed to work on macOS. So if you're going to take a hard line here, you should be ready to drop support for the OS entirely.

**Reply by tumic0** [reply]
I'm definitely ready. The day Apple decides what you can run on your computer not you, will be the last day of GPXSee for macOS. But while the screws are tightened every year by Apple, they had not yet the courage to close the macOS ecosystem like they did with iOS and the only reason is, there are still enough applications like GPXSee that refuse to submit to Apple...
